src/gui/AstroCalcDialog.hpp

```diff
@@ -203,6 +214,12 @@ private slots:
 	void saveTransits();
 	void setTransitCelestialBodyName();
 
+	//! Calculating lunar eclipses to fill the list.
```

Comment:
Please add any references here. "Algorithm taken from..." (or developed by yourself?)

Reply:
This part is just follow the same pattern from a few lines above it. I think you mean the AstroCalcDialog.cpp?

Reply:
The header file is the place for general developer documentation (Doxygen format). Literature information (if available) should go here. Inside the cpp file of course more details can be given in further comments.

Reply:
I hope I did right here. If not, please educate me. I'm still in the learning process.